cfme/scripting/quickstart.py

```diff
@@ -131,11 +131,12 @@
 
     if not REQUIRED_PACKAGES:
         print('{} {} not known. '
-              'Please ensure you have the required packages installed.')
+              'Please ensure you have the required packages installed.'.format(
```

Comment:
@ohiliazo Heads up, this `.format` was missed in #6685. Not trying to call you out in a mean way, just making you aware, and pointing out that reviewers (@RonnyPfannschmidt  and me) should have caught this first time around. Thanks @bsquizz for finding and fixing.